CONTRIBUTING.md

```diff
@@ -48,10 +48,23 @@ Please note that your contributions are shareable and remixable under such licen
 cannot be used for commercial purposes by Deck of Adventures users. As a contributor,
 you agree that being attributed via our list of contributors page satisfies the
 attribution requirements of the license applicable to your contribution to a page.
-Contributions are always welcome as github issues. GitHub pull requests should be
-reserved for major changes to at least one section, or minor changes across a many
+Contributions are always welcome as GitHub issues. GitHub pull requests should be
+reserved for major changes to at least one section, or minor changes across many
 sections (e.g., for terminology changes).
 
+<!--LOCK: Adding section for how we handle community contributions as Issues and the stages of consideration, development, playtesting, and integration into the core ruleset -->
+
+Contributions may take the form of suggested ideas or mechanics that could be
+implemented into the existing version. The Deck of Adventures admins reserve
+the right to choose which proposals and contributions are considered for
+integration, which are modified, adapted, playtested, and ultimately
+integrated into the current or future version.
+
+An Issue will be tagged as "Under Consideration", "In Development", "In
+Playtesting", and "Integrated" to establish where it is in the development
+cycle. Not all ideas will make it to the Integrated step, but every
+Contribution and comment from the Community is valued. 
```

Comment:
I think if we're leveraging this for public community visibility we'll want something a bit softer than the default "wontfix". The goal here is visibility, and transparency, so if we want to go with closed and not labeled "no thanks" as the way of highlighting "Integrated" that's fine, we'll just want to make sure the community knows this is how it's resolved and indicated what's resolved versus what's being passed over/disregarded. I was envisioning some status to indicate it was merged into the main rules so we can go back to the Issue submitter and say "congrats, your Issue was [INTEGRATED]. You can view the closed issue [here](link)"

Reply:
That sounds less like issue labels and more like a public project board where someone can see how their issue is moving though the integration process

Reply:
Hm this might be worth discussing further in that case. Part of why setting it up as Issue labels made sense to me to start is because we will want to be driving community members to engage with Issues - to add comments, create new issues, and keep tabs on outstanding issues they're passionate about or interested in. If we create a public project board it may help for visualizing the development process, but driving people to view that project board feels more passive "hey go to the project board so you can check out what we're working on" versus saying "hey go to Issues, add comments and submit your own so the community can keep updating and refining the game". 

Definitely open to a bigger discussion around how we want to approach this, I think starting with Issue labels and seeing how it goes from there might be a good starting point for heading into Beta, especially given that those issue labels can be folded into a project board for tagging/tracking in the future without us needing to make and maintain a full additional project board at this point when it might not be needed at this stage.